docs/docs/migration/v0.22.2.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 0
+title: v0.22.2
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.22.2, changes are required to use Ignite CLI v0.22.2. 
+---
+
+# Upgrading a blockchain to use Ignite CLI v0.22.2
+
+Ignite CLI v0.22.2 changed the GitHub username from "ignite-hq" to "ignite", which means the imports must be fixed to reflect this change.
+
+1. In your `go.mod` file find the require line for Ignite CLI that starts with `github.com/ignite-hq/cli` and is followed by a version.
+   It looks something like `github.com/ignite-hq/cli v0.22.0`, and replace it by `github.com/ignite/cli v0.22.2`.
+
+2. Make a bulk find and replace in the import statements for `github.com/ignite-hq/cli` to be replaced by `github.com/ignite/cli`.
```

Comment:
@ilgooz @tbruyelle should we add a command to make the replacements or leave it to the developer?

If we want to add a command like `find` & `sed` we should contemplate possible differences between the GNU and non GNU versions between linux and macOS.

Reply:
For me it's fine like that.

For the future, we can imagine a `ignite fix` command (just like there's a `go fix` command), dedicated to migrations like that.